package-native-zip.sh

```diff
@@ -103,6 +103,23 @@ echo "------------------------------------------------"
 echo ""
 zip -j dist/$PACKAGE_NAME dist/conduct dist/sandbox dist/shazar
 
+if [ "$UNAME_VALUE" = "Darwin" ]; then
+    echo "------------------------------------------------"
+    echo "Commands to create the Homebrew package pull request for $PACKAGE_NAME"
+    echo "------------------------------------------------"
+    echo ""
+
+    echo "Checkout the repo at https://github.com/typesafehub/homebrew-conductr and then issue"
```

Comment:
Looks like you also need to have `hub` installed and configured. I already did, so it just worked for me, but it might be worth noting here. You can install it with Homebrew (`brew install hub`) and IIRC it prompts for credentials when you first use it.

https://github.com/Homebrew/brew/blob/e022239608ca535fa3b56305eff8dd091b3fc008/Library/Homebrew/dev-cmd/bump-formula-pr.rb#L303-L304